inz-demo/id-card/Issues/address/description.xml

```diff
@@ -0,0 +1,34 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo ID Card Address</en>
+        <nl>Demo ID Card Address</nl>
+    </Name>
+    <ShortName>
+        <en>Address</en>
+        <nl>Address</nl>
+    </ShortName>
+    <SchemeManager>inz-demo</SchemeManager>
+    <IssuerID>id-card</IssuerID>
+    <CredentialID>address</CredentialID>
+    <Description>
+        <en>Full address</en>
+        <nl>Full address</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
+    <IssueURL>
+        <en>https://demo.irma.nz/id-card/</en>
+        <nl>https://demo.irma.nz/id-card/</nl>
+    </IssueURL>
+    <Attributes>
+        <Attribute id="fullAddress">
+            <Name>
+                <en>Full address</en>
+                <nl>Ailadres</nl>
```

Comment:
Let's leave `nl` values empty for now. I tried to use them for PBDF, but as it was not accurate, they asked to fix it or remove it. As we are not dutch speakers, better to remove them sooner.